cub/README.md

```diff
@@ -3,23 +3,23 @@
 
 CUB provides state-of-the-art, reusable software components for every layer
 of the CUDA programming model:
-- [<b><em>Device-wide primitives</em></b>](https://nvlabs.github.io/cub/group___device_module.html)
+- [<b><em>Device-wide primitives</em></b>](https://nvidia.github.io/cccl/cub/device_wide.html)
   - Sort, prefix scan, reduction, histogram, etc.
   - Compatible with CUDA dynamic parallelism
-- [<b><em>Block-wide "collective" primitives</em></b>](https://nvlabs.github.io/cub/group___block_module.html)
+- [<b><em>Block-wide "collective" primitives</em></b>](https://nvidia.github.io/cccl/cub/block_wide.html)
   - I/O, sort, prefix scan, reduction, histogram, etc.
   - Compatible with arbitrary thread block sizes and types
-- [<b><em>Warp-wide "collective" primitives</em></b>](https://nvlabs.github.io/cub/group___warp_module.html)
+- [<b><em>Warp-wide "collective" primitives</em></b>](https://nvidia.github.io/cccl/cub/warp_wide.html)
   - Warp-wide prefix scan, reduction, etc.
   - Safe and architecture-specific
 - [<b><em>Thread and resource utilities</em></b>](https://nvlabs.github.io/cub/group___util_io.html)
```

Comment:
I would prefer to just completely get rid of the broken links. Since we don't have a `module` for this group to link to right now, I'd suggest to just remove the link here.
```suggestion
- <b><em>Thread and resource utilities</em></b>
```